src/utils/get-organized-arguments.ts

```diff
@@ -283,12 +284,14 @@ export default async function getOrganizedArguments(): Promise<OrganizedArgument
 
 		// Validate data, if necessary
 		if (mergedSpec.data.accepts) {
+			const accepts =
+				typeof mergedSpec.data.accepts === 'function' ? await mergedSpec.data.accepts() : mergedSpec.data.accepts;
 			// @ts-expect-error: TypeScript is confused here...
```

Comment:
I believe I understand what's happening here.  `Array<string|number>` is equivalent to `(string|number)[]` not `string[] | number[]`, which seems to be why it's complaining.

Reply:
Hmm, I hadn't thought of that. I think right now the assumption is that it'll either be all one type. I'd have to look into whether this can be changed or not, though. If so, it'd be a different issue.